### ISSUE TITLE
'case' statement and semicolons

### ISSUE THREAD

**Issue by gamename** [skudriashev/bashlint#1]
Hi,

For some reason, the tool is flagging this (below) as 'W203 Trailing semicolon':

```
while getopts ":u:o:df" opt; do
  case ${opt} in
   u)
      url="${OPTARG}"
      echo "==> ${THIS_BASE} - Url to retrevie box/ovf is ${url}"
      ;; <----------Here 
   o)
      current_ovf_dir="${OPTARG}"
      echo "==> ${THIS_BASE} - Output directory will be ${current_ovf_dir}"
      ;;  <--------Here
```

Those are correct bash case statements.  What am I missing?

Thanks,
-T

**Comment by skudriashev** [comment]
@gamename thanks for the  bringing this up! I've committed the fix, please, verify. Let me know the result :)

**Comment by gamename** [comment]
The fix looks good. :)  Thanks for the quick turn-around.

Unfortunately, there may be another issue this uncovered.   I have several instances of 'find' in scripts which sometimes requires a '\;' at the end of the statement. Here is an example:

```
 bashlint --show-source
./build_boxes.sh:157:78: W203 Trailing semicolon
/usr/bin/find "${BUILD_PATH}" -type f -name 'kernel_report.txt' -exec cat {} \;
                                                                              ^
```

Its pretty minor, but wanted to let you know.  A backslash semicolon is valid in the case above.

Thanks again,
-Tennis

**Comment by skudriashev** [comment]
@gamename should be fixed now. Please, check. Let me know if any problems.

P.s. If you have any ideas for new checkers, please, tell me and I'll implement them :)

BR,
Stanislav.

**Comment by gamename** [comment]
Hi guy.

Been distracted by other problems.  I'm just now getting back to this and will let you know what I find out asap.

-T

**Comment by gamename** [comment]
Tested.  Looked good. :)

**Comment by skudriashev** [comment]
Awesome! Closing issue.
